docs/technical_reference/unit_models/anaerobic_digestor.rst

```diff
@@ -0,0 +1,129 @@
+
+.. figure:: ../../_static/unit_models/anaerobic_digestor.png
+    :width: 400
+    :align: center
+
+    Figure 1. Schematic representation of an anaerobic digestor
+
+.. index::
+   pair: watertap.unit_models.anaerobic_digestor;anaerobic_digestor
+
+.. currentmodule:: watertap.unit_models.anaerobic_digestor
+
+Degrees of Freedom
+------------------
+Aside from the inlet feed state variables (i.e. temperature, pressure, component flowrates), the ADM1 model has
+at least 5 degrees of freedom that should be fixed for the unit to be fully specified.
+
+Typically, the following variables are fixed, in addition to state variables at the inlet:
+    * cation concentration
+    * anion concentration
+    * liquid volume
+    * vapor volume
+    * liquid outlet temperature
+
+Control Volumes
+---------------
+
+This model has two separate 0D control volumes for the liquid and vapor phases.
+
+* Liquid
+* Vapor
+
+Ports
+-----
+
+This model provides three ports:
+
+* inlet
+* liquid_outlet
+* vapor_outlet
+
+Sets
+----
+.. csv-table::
+   :header: "Description", "Symbol", "Indices"
+
+   "Time", ":math:`t`", "[0]"
+   "Inlet/outlet", ":math:`x`", "['in', 'out']"
+   "Phases", ":math:`p`", "['Liq', 'Vap']"
+   "Components", ":math:`j`", "['H2O', 'S_su', 'S_aa', 'S_fa', 'S_va', 'S_bu', 'S_pro', 'S_ac', 'S_h2', 'S_ch4', 'S_IC', 'S_IN', 'S_I', 'X_c', 'X_ch', 'X_pr', 'X_li', 'X_su', 'X_aa', 'X_fa', 'X_c4', 'X_pro', 'X_ac', 'X_h2', 'X_I', 'S_cat', 'S_an', 'S_co2']"
+   "Ion", ":math:`j`", "['S_cat', 'S_an'] \  :sup:`*`"
+
+**Notes**
+ :sup:`*` Ion" is a subset of "Component" and uses the same symbol j.
+
+.. _AD_variables:
+
+Variables
+----------
+
+.. csv-table::
+   :header: "Description", "Symbol", "Variable Name", "Index", "Units"
+
+   "Liquid phase mass transfer term", ":math:`J`", "liquid_phase.mass_transfer_term_j", "[t]", ":math:`\text{kg/s}`"
+   "Liquid volume", ":math:`V_{ad, liq}`", "volume_liquid", "[t]", ":math:`\text{m}^3`"
+   "Vapor volume", ":math:`V_{ad, vap}`", "volume_vapor", "[t]", ":math:`\text{m}^3`"
+   "Total volume", ":math:`V_{ad}`", "volume_AD", "[t]", ":math:`\text{m}^3`"
+   "CO2 Henry's law coefficient", ":math:`K_{H, CO_{2}}`", "KH_co2", "None", ":math:`\text{kmol}\text{m}^{-3}\text{bar}^{-1}`"
+   "CH4 Henry's law coefficient", ":math:`K_{H, CH_{4}}`", "KH_ch4", "None", ":math:`\text{kmol}\text{m}^{-3}\text{bar}^{-1}`"
+   "H2 Henry's law coefficient", ":math:`K_{H, H_{2}}`", "KH_h2", "None", ":math:`\text{kmol}\text{m}^{-3}\text{bar}^{-1}`"
+   "Gas-liquid transfer coefficient", ":math:`K_{L}a`", "K_La", "None", ":math:`\text{day}^{-1}`"
+   "Friction parameter", ":math:`k_{p}`", "k_p", "None", ":math:`\text{m}^{3}\text{bar}^{-1}\text{day}^{-1}`"
+
+.. _AD_equations:
+
+Equations and Relationships
+---------------------------
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Outlet vapor phase pressure", ":math:`P_{vap, out} = P_{wat, sat} + /sum{j} P_{j, sat}`"
```

Comment:
@MarcusHolly The summation symbol (sigma) doesn't show up here. It's minor, but please add it to the tasklist in issue #909.